### ISSUE TITLE
Fixed - Unable to run jupyter notebook

### ISSUE THREAD

**Issue by arishsatheesan** [Xilinx/finn#1147]
I tried to run the commant './run_docker.sh notebook' and I receive the error:
"docker: Error response from daemon: driver failed programming external connectivity on endpoint amazing_ishizaka (d09773292be872dd03643b5ffc152c7321396b140ead72a89d0ccbf829bf8b9f): Error starting userland proxy: listen tcp4 0.0.0.0:8081: bind: address already in use."

![image](https://github.com/user-attachments/assets/8fc1f1b0-1819-4478-84f1-d6a84b24f0e7)

**Comment by Ba1tu3han** [comment]
Hello Dear @arishsatheesan,

Could you try to restart your Ubuntu system?

**Comment by arishsatheesan** [comment]
Hi,
I tried, but still the same issue.

**Comment by arishsatheesan** [comment]
Fixed it! The error was stating that a port 8081 is already in use. What I did was:
1. Type: sudo lsof -i :8081
    I got this:
    COMMAND PID   USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
    motion  931 motion    5u  IPv4  21958      0t0  TCP localhost:tproxy (LISTEN)
2. So, I had to stop the command 'motion'. Either one of the following can be used:
   sudo service stop motion (This didn't work for me.)
   sudo systemctl stop motion (This worked.)

 After this I ran './run-docker.sh notebook' and the notebook is opened.

Detailed discussion on this issue is here: https://stackoverflow.com/questions/69899466/error-starting-userland-proxy-listen-tcp4-0-0-0-080-bind-address-already-in
